Makefile

```diff
@@ -298,9 +312,9 @@ $(OUT_SHARED): $(OBJ_SHARED)
 
 static: $(OUT_STATIC)
 
-$(OUT_STATIC): $(OUT_SHARED)
```

Comment:
Won't this break the dependency tree? The static library depends on the shared library objects to create the archive. Did you test it both ways?